src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -266,6 +267,59 @@ public IEnumerable<ITagSpan<IClassificationTag>> GetTags(NormalizedSnapshotSpanC
                 }
             }
 
+            /// <summary>
+            /// Force computation of up to date tags.
+            /// This is called by the editor for scenarios such as print or rich text copy with classifications enabled.
+            /// Note that the editor will block the UI until results are returned.
+            /// </summary>
+            public IEnumerable<ITagSpan<IClassificationTag>> GetAllTags(NormalizedSnapshotSpanCollection spans, CancellationToken cancellationToken)
```

Comment:
this is really complicated code.  i'm very hesitant about this PR entirely.  

1. it feels like something we don't need to have.
2. i think it gives us false confidence in terms of how the apex test is working.
3. this is non trivial code to understand and maintain.